frameworks/JavaScript/ultimate-express/benchmark_config.json

```diff
@@ -0,0 +1,123 @@
+{
+  "framework": "ultimate-express",
+  "tests": [{
+    "default": {
+      "json_url": "/json",
+      "plaintext_url": "/plaintext",
```

Comment:
Only json and plaintext test here !!
It' the same with PostgreSQL. !!!

Reply:
I am not fully understanding this comment.